Sources/Basics/Sandbox.swift

```diff
@@ -15,8 +15,8 @@ import func TSCBasic.determineTempDirectory
 
 public enum SandboxNetworkPermission: Equatable {
     case none
-    case local(ports: [UInt8])
-    case all(ports: [UInt8])
+    case local(ports: [Int])
+    case all(ports: [Int])
```

Comment:
It could be, but the convention in the Swift networking libraries I've looked at is to use `Int`.   I believe it's to retain maximum generality in the API.   Practically, at the moment, we'll only ever see `UInt16`s.